includes/definitions/arris-apex.yaml

```diff
@@ -0,0 +1,11 @@
+os: arris-apex
+text: 'Arris Apex'
+type: network
+icon: arris
+mib_dir:
+    - arris
+over:
+    - { graph: '', text: '' }
```

Comment:
Isn't it any modem counter available that you could poll and display here ? In a CMTS this is usually something you would like to see displayed as the over graph.